_data/simple-icons.json

```
@@ -7465,6 +7465,16 @@
             "hex": "09009B",
             "source": "https://www.goindigo.in"
         },
+        {
+            "title": "Inductive Automation",
+            "hex": "435466",
+            "source": "https://inductiveautomation.com",
+            "guidelines": "https://inductiveautomation.frontify.com/d/PvUNchzTCkyj/inductive-automation-brand-guidelines",
+            "license": {
+                "type": "custom",
+                "url": "https://inductiveautomation.frontify.com/d/PvUNchzTCkyj/inductive-automation-brand-guidelines?#/legal/brand-usage-policy"
+            }
```

```suggestion
            "source": "https://brand.inductiveautomation.com",
            "guidelines": "https://brand.inductiveautomation.com"
```

This URL is what they have linked in the footer, redirects to that external website - looks cleaner and makes the connection clear. We also don't need to add a custom license since that information would be under guidelines and is linked above.